.github/workflows/staging.yml

```diff
@@ -0,0 +1,116 @@
+name: Staging
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  staging:
+    name: Website Staging
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - run: echo ::set-env name=PR_NUMBER::$(echo "$GITHUB_REF" | awk -F / '{print $3}')
+      shell: bash
+
+    - uses: actions/checkout@v2
+      with:
+        path: gh-actions
+        fetch-depth: 0
+
+    - name: Setup branch
+      run: |
+        cd gh-actions
+        git fetch origin $GITHUB_REF:testBranch
+        git checkout testBranch
+        
+    - name: Add Banner
+      run: |
+        cd gh-actions
+        cat >>"src/handlebars/partials/header.handlebars" <<-EOF
+        <div class="alert align-center">
+        <span class="closebtn" onclick="this.parentElement.style.display='none';">&times;</span>
+        This is a staging server, currently hosting <a class="light-link" href="https://github.com/AdoptOpenJDK/openjdk-website/pull/$PR_NUMBER"><var>PR $PR_NUMBER</var></a>
+        </div>
+        EOF
+
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '12'
```

Comment:
CAn we go higher?

Reply:
@karianna We have to match the production Node.js version for now: https://github.com/AdoptOpenJDK/openjdk-website/blob/master/.github/workflows/deploy-to-gh-pages.yml#L15-L17

As a separate PR I'll see if we can bump things up